"""Эта зараза str_func.py почему-то сохранилась в main"""
